src/components/Navigation/SearchBar/SearchBar.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <section class="np-search-bar">
+    <button type="button"
+            class="np-search-bar__toggle-list-view"
+            @click="changeFoodListView">
+      <ListIcon />
+    </button>
+
+    <div class="np-search-bar__input">
+      <SearchIcon class="np-search-bar__input-icon" />
+
+      <input class="np-search-bar__input-el"
+             type="search"
+             placeholder="Search"
+             v-model="filterVal"
+             />
+    </div>
+
+    <section class="np-search-bar__filter">
+      <div class="np-search-bar__filter-option"
+           @click="toggleOrderList">
+        <SwitchIcon class="np-search-bar__filter-option-icon"/>
+
+        <p class="np-search-bar__filter-option-text">
+          <span>{{orderVal.category}}: </span>{{orderVal.order}}
+        </p>
+
+        <button class="np-search-bar__filter-option-btn"
+                :class="isLiActive ? 'np-search-bar__filter-option-btn--active' : ''">
+
+          <DownArrowIcon />
+        </button>
+      </div>
+      <ul class="np-search-bar__filter-list"
+          :class="isLiActive ? 'np-search-bar__filter-list--active' : ''"
```

Comment:
skoro dwa razy używasz dynamicznych klas i robisz sprawdzenie `isLiActive ? someClassName : ''` to zrób z tego funkcję która przyjmuje someClassName jako parametr